src/lambda_function.py

```diff
@@ -54,3 +56,39 @@ def check_prime(num):
             return False
         sq_root += 1
     return True
+
+
+def sync_ddb_table(source_ddb, destination_ddb):
+    response = source_ddb.scan(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `scan` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `ExclusiveStartKey`, `IsTruncated`, `LastEvaluatedKey`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)